examples/order_manager/order_service/tests/Config.toml

```diff
@@ -0,0 +1,4 @@
+[[ballerina.auth.users]]
```

Comment:
Better to add all the configurations to `Config.toml`. TOPIC, LISTENER_PORT etc

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/278/commits/0597f377c61e2a791d4931916e97ae3b03e4c582